docs/usage/INVENTORY_SYNC.md

```diff
@@ -4,122 +4,169 @@ Utility which provides API for building CTP inventory update actions and invento
+    - [Running the sync](#running-the-sync)
+  - [Build all update actions](#build-all-update-actions)
+  - [Build particular update action(s)](#build-particular-update-actions)
+- [Caveats](#caveats)
 
-### Build all update actions
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-<!-- TODO: Probably #14 affects inventory sync as well. Ensure before providing the code snippet. -->
+## Usage
 
-### Build particular update action(s)
+### Sync list of inventory entry drafts
 
-To build the update action for changing inventory quantity:
+<!-- TODO - GITHUB ISSUE#138: Split into explanation of how to "sync from project to project" vs "import from feed"-->
 
+#### Prerequisites
+1. The sync expects a list of `InventoryEntryDraft` objects that have their `sku` fields set,
+   otherwise the sync will trigger an `errorCallback` function set by the user (more on it can be found down below in the options explanations).
+
+2. Every inventory entry may have a reference to a supply `Channel` and a reference to the `Type` of its custom fields. These
+   references are matched by their `key`. Therefore, in order for the sync to resolve the actual ids of those references,
+   their `key`s has to be supplied in the following way:
+   - Provide the `key` value on the `id` field of the reference. This means that calling `getId()` on the
+   reference would return its `key`. 
+     
+        **Note**: This library provides you with a utility method 
+         [`replaceInventoriesReferenceIdsWithKeys`](https://commercetools.github.io/commercetools-sync-java/v/v1.0.0-M14/com/commercetools/sync/inventories/utils/InventoryReferenceReplacementUtils.html#replaceInventoriesReferenceIdsWithKeys-java.util.List-)
+         that replaces the references id fields with keys, in order to make them ready for reference resolution by the sync:
+         ````java
+         // Puts the keys in the reference id fields to prepare for reference resolution
+         final List<InventoryEntryDraft> inventoryEntryDrafts = replaceInventoriesReferenceIdsWithKeys(inventoryEntries);
+         ````
+     
+3. It is an important responsibility of the user of the library to instantiate a `sphereClient` that has the following properties:
+    - Limits the amount of concurrent requests done to CTP. This can be done by decorating the `sphereClient` with 
+   [QueueSphereClientDecorator](http://commercetools.github.io/commercetools-jvm-sdk/apidocs/io/sphere/sdk/client/QueueSphereClientDecorator.html) 
+    - Retries on 5xx errors with a retry strategy. This can be achieved by decorating the `sphereClient` with the 
+   [RetrySphereClientDecorator](http://commercetools.github.io/commercetools-jvm-sdk/apidocs/io/sphere/sdk/client/RetrySphereClientDecorator.html)
+   
+   You can use the same client instantiating used in the integration tests for this library found 
+   [here](/src/main/java/com/commercetools/sync/commons/utils/ClientConfigurationUtils.java#L45).
```

Comment:
`If you have no special requirements on sphere client creation then you can use "ClientConfigurationUtils.createClient(...)" which applies best practices already.`

Reply:
21ce209